ci-operator/step-registry/baremetalds/e2e/test/baremetalds-e2e-test-commands.sh

```diff
@@ -43,4 +43,5 @@ ssh "${SSHOPTS[@]}" "root@${IP}" tar -czf - /tmp/artifacts | tar -C "${ARTIFACT_
 
 set -e
 echo "### Done! (${rv})"
+touch $SHARED_DIR/e2e_test_complete
```

Comment:
@stbenjam do you think it would be useful to gather logs in case of conformance tests failure?

Reply:
Yea a must-gather is definitely useful

Reply:
Yea a must-gather is definitely useful